package.json

```diff
@@ -2656,19 +2656,23 @@
                 },
                 {
                     "id": "dockerNetworks",
-                    "name": "%vscode-docker.views.dockerNetworks%"
+                    "name": "%vscode-docker.views.dockerNetworks%",
+                    "visibility": "collapsed"
                 },
                 {
                     "id": "dockerVolumes",
-                    "name": "%vscode-docker.views.dockerVolumes%"
+                    "name": "%vscode-docker.views.dockerVolumes%",
+                    "visibility": "collapsed"
                 },
                 {
                     "id": "vscode-docker.views.dockerContexts",
-                    "name": "%vscode-docker.views.dockerContexts%"
+                    "name": "%vscode-docker.views.dockerContexts%",
+                    "visibility": "collapsed"
                 },
                 {
                     "id": "vscode-docker.views.help",
-                    "name": "%vscode-docker.views.help%"
+                    "name": "%vscode-docker.views.help%",
+                    "visibility": "collapsed"
```

Comment:
Morty did not say collapse Help by default--can you double-check with him on that?

Reply:
I'll check and argue with him if he wants it open by default. It's so annoying. :smile:

Reply:
He said keep the Help panel open but at the bottom. Made it so.